src/test/java/com/wikia/webdriver/testcases/desktop/adstests/TestAdsUapHiVi.java

```diff
@@ -3,6 +3,7 @@
 import com.wikia.webdriver.common.core.Assertion;
 import com.wikia.webdriver.common.core.WikiaWebDriver;
 import com.wikia.webdriver.common.core.elemnt.JavascriptActions;
+import com.wikia.webdriver.common.dataprovider.ads.AdsDataProvider;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'com.wikia.webdriver.common.dataprovider.ads.AdsDataProvider'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AUselessImportCheck)